include/mvdtool/bits/mvd3_misc.hpp

```diff
@@ -206,6 +213,27 @@ inline std::vector<std::string> MVD3File::getSynapseClass(const Range & range) c
 }
 
 
+inline std::vector<TSV::TSVInfo> MVD3File::getTSVInfo(const Range& range) const {
```

Comment:
Hi @iomaganaris 
I think get-tsv() should be internal. TSV stands just for tab-separated and it has no semantics per se, worth to mvd fields.
I thought we would provide a unified public API which is, when the user requests, e.g. the etypes, it will 
 - Sonata: read from the dedicated field
 - Sonata+tsv: read from tsv
 - mvd3 alone: fail
 - mvd3+tsv: read from tsv

But maybe this is just the way I imagined which might not be the best.
@matz-e @pramodk could you please comment here?

Reply:
Ah okay. In my case I have imagined it differently. I thought that in `Neurodamus` it is just needed to get the `TSVInfo` struct for every cell and then handle it like it does now with the `METypeInfoItem`.

Reply:
Yes, that could be an option but without great impact because we can also read the TSV directly. The great value comes from asking MVDtool the fields we need, and MVDtool figure it out for us

Reply:
Yes, I definitely see your point on that :+1: